portal-web/test/functional/com/liferay/portalweb/paths/portlet/search/SearchTuning.path

```diff
@@ -33,6 +33,11 @@
 	<td>//div[@data-testid='add-result-modal']//input[contains(@aria-label,'Search')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>RESULT_RANKINGS_ADD_RESULT_TOP_RESULT</td>
+	<td>//li[contains(@class,'list-group-item')]</td>
+	<td></td>
+</tr>
```

Comment:
This path actually matches all results in the "Add Results" modal.

Try creating a path like `SearchPage#RESULT_TITLE_SPECIFIC_ORDER`. This path takes the specific order of the result so it's reusable in other scenarios.